pkg/util/k8sutil/pod_util.go

```diff
@@ -70,10 +70,10 @@ func containerWithRequirements(c v1.Container, r v1.ResourceRequirements) v1.Con
 
 func newEtcdProbe(isSecure bool) *v1.Probe {
 	// etcd pod is alive only if a linearizable get succeeds.
```

Comment:
Update this comment as well.
```
// etcd pod is healthy only if it can participate in consensus
```

Reply:
@Seth-Miller can you address the above comment.